test/gutenberg-test-themes/block-bindings/style.css

```diff
@@ -0,0 +1,15 @@
+/*
+Theme Name: Block Bindings
+Theme URI: https://github.com/wordpress/gutenberg/
+Author: the WordPress team
+Description: Block bindings test theme.
+Requires at least: 5.3
```

Comment:
I think it requires at least: 6.7.

Reply:
Addressed in this commit: https://github.com/WordPress/gutenberg/pull/65526/commits/c7457d0f29f0759993ad85794721ab90896ac351